packages/slice-machine/components/AppLayout/Navigation/Menu/Desktop.tsx

```diff
@@ -119,18 +100,30 @@ const UpdateInfo: React.FC<{
 const Desktop: React.FunctionComponent<{ links: LinkProps[] }> = ({
   links,
 }) => {
-  const { warnings, configErrors, updateVersionInfo } = useSelector(
-    (store: SliceMachineStoreType) => ({
+  const { warnings, configErrors, updateVersionInfo, versionsSeen } =
+    useSelector((store: SliceMachineStoreType) => ({
       warnings: getWarnings(store),
       configErrors: getConfigErrors(store),
       updateVersionInfo: getUpdateVersionInfo(store),
-    })
-  );
+      versionsSeen: getVersionsTheUserKnowsAbout(store),
+    }));
+
+  const { viewedUpdates } = useSliceMachineActions();
+
+  const userSeenUpdates =
+    versionsSeen &&
+    versionsSeen.patch === updateVersionInfo.availableVersions.patch &&
+    versionsSeen.minor === updateVersionInfo.availableVersions.minor &&
+    versionsSeen.major === updateVersionInfo.availableVersions.major
+      ? true
```

Comment:
simplify this into : 

````
versionsSeen &&
    versionsSeen.patch === updateVersionInfo.availableVersions.patch &&
    versionsSeen.minor === updateVersionInfo.availableVersions.minor &&
    versionsSeen.major === updateVersionInfo.availableVersions.major
````

Reply:
k, the `userSeenUpdates` might be a string